docs/reference/nuctl/nuctl.md

```diff
@@ -0,0 +1,83 @@
+# Nuctl
+
+## About
+
+Nuctl is Nuclio command-line interface that provide all nuclio-features from your terminal
+
+## Download
+
+To install nuctl all you need is to simply visit Nuclio [releases page](https://github.com/nuclio/nuclio/releases)
+And download the matched binary to your platform (e.g.: `darwin` if you have `macOS`)
+
+If you are adventurous enough, you can use the following snippet
+```bash
+curl -s https://api.github.com/repos/nuclio/nuclio/releases/latest \
+			| grep -i "browser_download_url.*nuctl.*$(uname)" \
+			| cut -d : -f 2,3 \
+			| tr -d \" \
+			| wget -O nuctl -qi - && chmod +x nuctl
+```
+
+## Use
+
+Once downloaded, an informative help section is available for each command using `nuctl --help`
+
+## Running platform
+
+Nuctl will automatically identify its running platform, whether it is Docker or Kubernetes.
+
+To ensure you run Nuctl against a specific platform, use `--platform kube` for Kubernetes or `--platform local` for docker
+
+### Docker
+
+Ensure your Docker daemon is running, run  `docker version` with the same user that will execute Nuctl commands
+
+An example of function deployment using nuctl:
```

Comment:
This example should be a section with sub sections for the different operations. It's long and has multiple parts / actions.
Also, I think since you're linking somewhere else for a k8s example, maybe it's best to move the nuctl docker example to it's own file as well, so to keep this readme short and "balanced" between the platform kinds

Reply:
moved